package.json

```diff
@@ -23,14 +23,14 @@
   },
   "homepage": "https://github.com/Pencroff/hapi-waterline",
   "dependencies": {
-    "lodash": "^2.4.1",
-    "require-dir": "^0.1.0",
-    "waterline": "^0.10.14"
+    "lodash": "^4.17.4",
```

Comment:
In code was used `let` from ES6, better then to describe minimal node version like in hapi package.json:
```
"engines": {
    "node": ">=8.9.0"
  },
```

Reply:
Not sure what you mean

Reply:
The words like `let`, `const`, `class`, etc. supported just from 6th version of Node, in 5th it will be exception. So if you used the new syntax, not 100% supported ES5, it's required restrictions on installation step. To prevent users from runtime exception if they are used old version of Node (in any case)

Reply:
Should be fixed not

Reply:
now